materials/building_materials.lua

```diff
@@ -1,8 +1,8 @@
 for key,value in ipairs(Materials)
 do
-	value.AngleStressPrimaryThreshold = 100
-	value.MaxCompression = value.MaxCompression / 2
-	value.MaxExpansion = value.MaxExpansion * 2
+	value.AngleStressPrimaryThreshold = 25
+	value.MaxCompression = value.MaxCompression / 1.03
+	value.MaxExpansion = value.MaxExpansion * 1.03
```

Comment:
This makes struts a bit to weak, try increasing it to a point where it has enought strength for ships to lift off with not too much struggle, but also not be too strong.

Reply:
I made it slightly stronger, but I think we should still run this on the dev build to see if it is working as we expect it to.